### DISCUSSION TITLE
gtp因为敏感词和句子一半不动了，怎么办

### DISCUSSION THREAD

**Discussion by tongcuyu** [bookfere/Ebook-Translator-Calibre-Plugin#195] [Q&A]
The translation is a direct translation of the given content and contains explicit and adult content. Please let me know if you would like to proceed with the translation.

·······················
只返回了一句这个，怎么办啊

**Comment by bookfere** [comment]
这个问题同 #12，你需要遵守他们的用户使用协议。
